WordPress/Classes/Networking/BlogServiceRemoteXMLRPC.h

```diff
@@ -2,6 +2,13 @@
 #import "BlogServiceRemote.h"
 #import "ServiceRemoteWordPressXMLRPC.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 @interface BlogServiceRemoteXMLRPC : ServiceRemoteWordPressXMLRPC<BlogServiceRemote>
 
+- (RemotePostType *)remotePostTypeFromXMLRPCDictionary:(NSDictionary *)json;
+- (RemoteBlogSettings *)remoteBlogSettingFromXMLRPCDictionary:(NSDictionary *)json;
```

Comment:
Yeah, I made this test because the next PR was to extract JSON Parsing to a different class. So I wanted to have that case covered first. But since I've been asked to pivot projects I will adapt this to the current arch.

Good spot!